src/CoreWCF.WebHttp/src/CoreWCF/Channels/StreamBodyWriter.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Xml;
+
+namespace CoreWCF.Channels
+{
+    public abstract class StreamBodyWriter : BodyWriter
+    {
+        // TODO: Can this be removed?
+        // if isQuirkedTo40Behavior = true, does not try to write out <Binary> tags 
```

Comment:
Yes, this can be removed. We don't need to maintain 4.0 behavior

Reply:
This is addressed in c730cd55a45faa975e61431f0e0145dbed592901.